validation/Cargo.toml

```diff
@@ -18,7 +18,7 @@ availability_store = { package = "polkadot-availability-store", path = "../avail
 parachain = { package = "polkadot-parachain", path = "../parachain" }
 polkadot-primitives = { path = "../primitives" }
 polkadot-erasure-coding = { path = "../erasure-coding" }
-polkadot-runtime = { path = "../runtime" }
+polkadot-runtime = { path = "../runtime/polkadot" }
```

Comment:
Internally the runtime is used to get the `NEW_HEADS_INDENTIFIER`. This identifier should be moved to primitives and the reference to the runtime should be removed here.